go.mod

```diff
@@ -0,0 +1 @@
+module github.com/russross/blackfriday/v2
```

Comment:
I'm not sure that the `/v2` suffix belongs here. I think it's expected to come from the tag version (e.g., tag [v2.0.0](https://github.com/russross/blackfriday/releases/tag/v2.0.0) would have the module-aware import path `github.com/russross/blackfriday/v2`). Please correct me if I'm wrong.